settings.py

```diff
@@ -89,11 +89,7 @@
     os.path.join(FILEPATH, 'locale'),
 )
 
-# Do not change, there's a weird bug with Django 1.7 that requires this to be bower_components when using
-# collectstatic
-BOWER_COMPONENTS_ROOT = os.path.join(FILEPATH, 'bower_components')
-
-BOWER_PATH = '/usr/local/bin/bower'
+BOWER_COMPONENTS = os.path.join(FILEPATH, 'bower_components', 'bower_components')
```

Comment:
realize it's awkward to have `bower_components/bower_components` but was hoping to make the switch easier for people by keeping the folder the same